core/src/test/java/org/mule/registry/TestDiscoverableObject.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.registry;
+
+public interface TestDiscoverableObject
```

Comment:
Can be TestDiscoverableObject and TestDiscoverableObjectImpl inner classes of SpiServiceRegistryTestCase?